store/mongo/datastore_mongo_cleanup.go

```diff
@@ -138,22 +137,25 @@ func (db *DataStoreMongo) DeleteBrokenAuthSets(dbName string) error {
 		return err
 	}
 
-	// delete authsets for non-exisitent devices
+	// delete authsets for non-existent devices
 	for _, as := range authSets {
-		_, err := collAuthSets.DeleteOne(nil, model.AuthSet{Id: as})
+		_, err := collAuthSets.DeleteOne(context.TODO(), model.AuthSet{Id: as})
 		if err != nil {
 			return errors.Wrapf(err, "database %s, failed to delete authentication sets", dbName)
 		}
 		// Attempt to delete token (may have already expired).
-		collTokens.DeleteOne(nil, bson.M{"_id": as})
+		_, _ = collTokens.DeleteOne(context.TODO(), bson.M{"_id": as})
```

Comment:
Define the context outside of the for loop and reuse it.
We usually use context.Background in this case, instead of context.TODO.

Reply:
Noted. I used `TODO` because is what the docs suggest to use instead of `nil`. I'll rework.